src/components/ProbeCard/ProbeCard.tsx

```diff
@@ -14,7 +14,7 @@ import { ProbeLabels } from './ProbeLabels';
 import { ProbeStatus } from './ProbeStatus';
 
 export const ProbeCard = ({ probe }: { probe: ExtendedProbe }) => {
-  const canEdit = useCanEditProbe(probe);
+  const { canWriteProbes } = useCanEditProbe(probe);
```

Comment:
Do we need to bring in `canReadProbes` for the alternative action? 🤔

Reply:
Read restrictions are enforced in the `includes` section of the `plugin.json` [file](https://github.com/grafana/synthetic-monitoring-app/blob/1ec89df9a78784c68f84d99df4b630fe052f65bf/src/plugin.json#L38) where we set the required permissions to load the respective urls, so without those actions, the pages can't be accessed. See also https://github.com/grafana/synthetic-monitoring-app/pull/986#discussion_r1859223153